lightning/src/ln/invoice_utils.rs

```diff
@@ -1117,32 +1084,16 @@ mod test {
 		connect_blocks(&nodes[2], CHAN_CONFIRM_DEPTH - 1);
 		confirm_transaction_at(&nodes[0], &tx, conf_height);
 		connect_blocks(&nodes[0], CHAN_CONFIRM_DEPTH - 1);
-		let as_channel_ready = get_event_msg!(
-			nodes[2],
-			MessageSendEvent::SendChannelReady,
-			nodes[0].node.get_our_node_id()
-		);
-		nodes[2].node.handle_channel_ready(
-			nodes[0].node.get_our_node_id(),
-			&get_event_msg!(
-				nodes[0],
-				MessageSendEvent::SendChannelReady,
-				nodes[2].node.get_our_node_id()
-			),
-		);
-		get_event_msg!(
-			nodes[2],
-			MessageSendEvent::SendChannelUpdate,
-			nodes[0].node.get_our_node_id()
-		);
-		nodes[0].node.handle_channel_ready(nodes[2].node.get_our_node_id(), &as_channel_ready);
-		get_event_msg!(
-			nodes[0],
-			MessageSendEvent::SendChannelUpdate,
-			nodes[2].node.get_our_node_id()
-		);
-		expect_channel_ready_event(&nodes[0], &nodes[2].node.get_our_node_id());
-		expect_channel_ready_event(&nodes[2], &nodes[0].node.get_our_node_id());
+		let as_channel_ready =
```

Comment:
Changing `as_channel_ready` from the message that `C` omits to send to `A` to the message that `A` omits to send `C` seems consistent with naming elsewhere in the codebase (at least the majority of cases) 👌